### PR TITLE
Fix arguments ordering for assertions in tests

### PR DESCRIPTION
**Pull request by alextrapper**
Fix for found asserts with wrong arguments (actual and expected) order.
Please review.

### CODE REVIEW

**Comment by IlyaMokin** [comment]
Thank you, looks good! Merging the chages to master

**Comment by IlyaMokin** [comment]
in some asserts arguments were mixed up, I've reverted the merge, could you check it please.
https://aistrainee2020.slack.com/archives/G015BT3P163/p1594192727315400

**Comment by alextrapper** [comment]
Shure

File: test/02-numbers-tests.js [hunk 1]
```diff
@@ -7,97 +7,97 @@ it.optional = require('../extensions/it-optional');
...
     });
 
     it.optional('toNumber should convert any value to number or return the default', function() {
-        assert.equal(0, tasks.toNumber(null, 0));
-        assert.equal(0, tasks.toNumber('test', 0));
-        assert.equal(1, tasks.toNumber('1', 0));
-        assert.equal(42, tasks.toNumber(42, 0));
-        assert.equal(42, tasks.toNumber(new Number(42), 0));
-        assert.equal(-1, tasks.toNumber(undefined, -1));
+        assert.equal(tasks.toNumber(null, 0), 9);
```
Comment (alextrapper) [inline test/02-numbers-tests.js:96]: Here was a typo

**Pull request merged**